azurelinuxagent/ga/exthandlers.py

```diff
@@ -935,11 +935,25 @@ def collect_ext_status(self, ext):
 
         return ext_status
     
+    def get_ext_handling_status(self, ext):
+        seq_no, ext_status_file = self.get_status_file_path(ext)
+        if seq_no < 0 or ext_status_file is None:
+            return None
+
+        # Missing status file is considered a non-terminal state here
+        # so that extension sequencing can wait until it becomes existing
```

Comment:
This will break existing extensions.

Reply:
can you give me an example of breaking case?